src/main/frontend/handler/repo.cljs

```diff
@@ -534,6 +534,25 @@
   (state/set-current-repo! repo)
   (db/start-db-conn! nil repo option))
 
+; Add translate function t in src/main/frontend/context/i18n.cljs without shortcut-dict/dict 
+; to avoid circular dependency
```

Comment:
```suggestion
; to avoid circular dependency
; TODO: Remove copied functions once circular dependency is resolved
```
It's not great that we have this copied code. Want to make this an explicit TODO so we address this soon

Reply:
I think that is better too. Thanks.